Makefile

```diff
@@ -137,21 +118,26 @@ create-cluster:
 delete-cluster:
 	k3d cluster list $(CLUSTER_NAME) 2>&- && k3d cluster delete $(CLUSTER_NAME)
 
+.ONESHELL:
 wait:
 	@echo "-----------------------------------------------------------"
 	@echo "--- Please wait until all services are in running state ---"
 	@echo "-----------------------------------------------------------"
-	@kubectl get -n $(NAMESPACE) --watch pods
-
-.provision-guacamole:
```

Comment:
The idea was to `touch .provision-guacamole` at the end of this step. I suppose that has disappeared.

Reply:
`touch .provision-guacamole` was a good idea until we dropped the persistent volume for the guacamole database. Every time the database Pod got restarted (which happens on new rollout and can happen after a new deployment when there were changes), the database structure was gone (no persistence). Therefore, I changed it to the "deploy always" mode. When the database structures are already there, postgres skips the creation.